### PR TITLE
Updating ADB box version to 1.8.0

### PR DESCRIPTION
**Pull request by LalatenduMohanty**
Signed-off-by: Lalatendu Mohanty lmohanty@redhat.com

### CODE REVIEW

**Comment by LalatenduMohanty** [comment]
#build

**Comment by bexelbie** [comment]
LGTM

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,8 @@
 # Changelog
 
-## Unreleased
+## v1.8.0 Apr 4, 2016
```
Comment (navidshaikh) [inline CHANGELOG.md]: should the date be updated?

**Comment by praveenkumar** [comment]
LGTM

**Pull request merged**